region/maridia/outer.json

```diff
@@ -1784,14 +1784,40 @@
                   "requires": [
                     "canSuitlessMaridia",
                     "HiJump",
-                    "canUseFrozenEnemies"
+                    "canTrickyUseFrozenEnemies"
+                  ]
+                },
+                {
+                  "name": "Main Street Climb with Ice and Spring Ball",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canSpringBallJumpMidAir",
+                    "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      "canCrazyCrabClimb",
```

Comment:
I'd call this either not crazy crab climb or not notable.  Its not that hard of a crab freeze.  You can freeze it right as it passes a corner on the wall so you're not going to fall through the crab.  But balancing the fish freeze and the two crab freezes are something.